default.yml

```diff
@@ -88,6 +88,9 @@ Metrics/ParameterLists:
 Naming/VariableNumber:
   EnforcedStyle: snake_case
 
+Performance/Casecmp:
```

Comment:
```suggestion
# Do not enforce using `str.casecmp('foo') == 0` instead of
# `str.downcase == 'foo'` for performance reasons.
Performance/Casecmp:
```